packages/p2p-v2/tailwind.config.js

```diff
@@ -0,0 +1,11 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+    content: ['./src/**/*.{js,jsx,ts,tsx}'],
+    theme: {
+        extend: {},
+    },
+    plugins: [],
+    corePlugins: {
+        preflight: false,
```

Comment:
This needs to be disabled, otherwise Tailwind will inject some default stylings that causes issues in the Deriv.app header stylings to be overriden